### PR TITLE
F#1 package skeleton

### LINKED ISSUE #1: Create package skeleton

### PR DESCRIPTION
**Pull request by toliver**
fixes #1

### CODE REVIEW

File: optoforce/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,188 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(optoforce)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  geometry_msgs
+  python-serial
```
Comment (ugocupcic) [inline optoforce/CMakeLists.txt]: I don't think this goes in there it's a rosdep not a ros pacakge

File: optoforce/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# optoforce
+
+ROS driver for the Optoforce sensor.
```
Comment (ugocupcic) [inline optoforce/README.md]: put an url to optoforce?

File: optoforce/package.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package>
+  <name>optoforce</name>
+  <version>0.0.1</version>
+  <description>optoforce contains a drover for the Optoforce sensor</description>
```
Comment (ugocupcic) [inline optoforce/package.xml]: driver

File: optoforce/CMakeLists.txt (hunk 1 above)
Comment (toliver) [inline optoforce/CMakeLists.txt]: Good point

File: optoforce/package.xml (hunk 3 above)
Comment (toliver) [inline optoforce/package.xml]: drover

File: optoforce/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+## Declare things to be passed to dependent projects
+## INCLUDE_DIRS: uncomment this if you package contains header files
+## LIBRARIES: libraries you create in this project that dependent projects also need
+## CATKIN_DEPENDS: catkin_packages dependent projects also need
+## DEPENDS: system dependencies of this project that dependent projects also need
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES optoforce
+#  CATKIN_DEPENDS geometry_msgs rospy
+#  DEPENDS system_lib
```
Comment (ugocupcic) [inline optoforce/CMakeLists.txt]: python-serial?

**Pull request merged**